executor/point_get.go

```diff
@@ -584,13 +586,10 @@ func tryDecodeFromHandle(tblInfo *model.TableInfo, schemaColIdx int, col *expres
 		chk.AppendInt64(schemaColIdx, handle.IntValue())
 		return true, nil
 	}
-	if types.NeedRestoredData(col.RetType) {
-		return false, nil
-	}
 	// Try to decode common handle.
 	if mysql.HasPriKeyFlag(col.RetType.Flag) {
 		for i, hid := range pkCols {
-			if col.ID == hid && notPKPrefixCol(hid, prefixColIDs) {
```

Comment:
The argument `prefixColIDs` is not used anymore. We can remove it from the signature.